pureconfig/src/main/scala/com/avast/clients/rabbitmq/pureconfig/PureconfigImplicits.scala

```diff
@@ -7,32 +7,13 @@ import _root_.pureconfig.generic.semiauto._
 import cats.data.NonEmptyList
 import com.avast.clients.rabbitmq.api.DeliveryResult
 import com.avast.clients.rabbitmq.api.DeliveryResult._
-import com.avast.clients.rabbitmq.{
-  AutoBindExchangeConfig,
-  AutoBindQueueConfig,
-  AutoDeclareExchangeConfig,
-  AutoDeclareQueueConfig,
-  BindArgumentsConfig,
-  BindExchangeConfig,
-  BindQueueConfig,
-  ConsumerConfig,
-  CredentialsConfig,
-  DeclareArgumentsConfig,
-  DeclareExchangeConfig,
-  DeclareQueueConfig,
-  ExchangeType,
-  NetworkRecoveryConfig,
-  ProducerConfig,
-  ProducerPropertiesConfig,
-  PullConsumerConfig,
-  RabbitMQConnectionConfig,
-  RecoveryDelayHandlers
-}
+import com.avast.clients.rabbitmq.{AddressResolverType, AutoBindExchangeConfig, AutoBindQueueConfig, AutoDeclareExchangeConfig, AutoDeclareQueueConfig, BindArgumentsConfig, BindExchangeConfig, BindQueueConfig, ConsumerConfig, CredentialsConfig, DeclareArgumentsConfig, DeclareExchangeConfig, DeclareQueueConfig, ExchangeType, NetworkRecoveryConfig, ProducerConfig, ProducerPropertiesConfig, PullConsumerConfig, RabbitMQConnectionConfig, RecoveryDelayHandlers}
```

Comment:
Our formatters obviously don't agree with each other, so I propose to change this huge import to `import com.avast.clients.rabbitmq.{pureconfig => _, _}`.